paasta_tools/paasta_metastatus.py

```diff
@@ -601,7 +619,7 @@ def main():
         mesos_state=mesos_state,
     )
     metrics = get_mesos_stats()
-    mesos_metrics_status = get_mesos_metrics_health(mesos_metrics=metrics)
+    mesos_metrics_status = get_mesos_metrics_health(mesos_metrics=metrics, mesos_state=mesos_state)
```

Comment:
We don't have to hit the master state endpoint - we could hit the state endpoint for each slave. It'll mean more work here, as we'll have to send the request to *every* slave in the cluster, and accumulate the results, but it does mean that we can avoid hitting the expensive state endpoint on the master. What do you think?